### PR TITLE
Create Designer Image

### PR DESCRIPTION
**Pull request by AlgoRando**
4VAXLOZBIJQTF6PU7FCGEO3HQYAGYIZAFPSN3DQ2CASSPACOTUW7NQCCVY

### CODE REVIEW

File: public/img/og/Designer Image [hunk 1]
```diff
@@ -0,0 +1 @@
+![image](https://user-images.githubusercontent.com/99285834/153058601-c7b5f628-29ae-437d-9f2f-6563d6bd183a.jpeg)
```
Comment (scholtz) [inline public/img/og/Designer Image]: can you please include in the commit the image itself, not the link please?

Comment (AlgoRando) [reply public/img/og/Designer Image]: It should now show the image.

**Comment by scholtz** [comment]
https://algoexplorer.io/tx/6MSDV5HADDI3RL3QQHLQXKAISVKFIJ326TNPTWZPVVOAMFUFEE5A

File: public/img/og/Designer Image (hunk 1 above)
Comment (scholtz) [reply public/img/og/Designer Image]: i still do not see it as a file

**Comment by AlgoRando** [comment]
I have created a file in the directory - Vote Logo.jpeg

This is showing when I view it.  And apologies, I am not familiar with GitHub.

File: public/img/og/Designer Image (hunk 1 above)
Comment (AlgoRando) [reply public/img/og/Designer Image]: Please see new comment.

**Pull request closed without merging**